resources/adoptopenjdk_install.rb

```diff
@@ -0,0 +1,202 @@
+#
+# Cookbook:: java
+# Resource:: adoptopenjdk_install
+#
+# Based on oracle_install.rb by Bryan W. Berry (<bryan.berry@gmail.com>)
+#
+
+resource_name :adoptopenjdk_install
+
+default_action :install
+
+property :url, String
+property :checksum, String, regex: /^[0-9a-f]{32}$|^[a-zA-Z0-9]{40,64}$/
+property :md5, String, regex: /^[0-9a-f]{32}$|^[a-zA-Z0-9]{40,64}$/
```

Comment:
while the other resources use `md5` I wonder if its worth creating this new one with a `sha256` or higher to reduce the likelihood of collisions @tas50 thoughts?

Reply:
we should probably go sha256 on anything new